File: cs50_x/week6/me/compare.c
// Compares two numbers. 

#include <stdio.h>

int main(void)
{
    // Gets input from standard input.
    int x, y;
    printf("x: ");
    scanf("%d", &x);
    printf("y: ");
    scanf("%d", &y);

    // Prints output to standard output.
    if (x > y)
        puts("x is greater than y.");
    else if (x < y)
        puts("y is greater than x.");
    else
        puts("x is equal to y.");
}

File: cs50_x/week6/me/compare.py
# Compares two numbers.

def main():
    # Gets input from standard input.
    x = int(input("x: "))
    y = int(input("y: "))

    # Prints output to standard output.
    if x > y:
        print("x is greater than y.")
    elif x < y:
        print("y is greater than x.")
    else:
        print("x is equal to y.")

main()